wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Annual details: change year displayed

Fixes #11908 

This adds a date bar to Insights Annual details. When the year is changed via the date bar, the displayed data updates accordingly.

NOTE: accompanying WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/162

To test:
- Go to Insights _This Year_. Select `View more`.
  - Verify the most recent year's data is displayed. This should match what is displayed on Insights.
  - Most recent year = the most recent year the site has data for. (can be confirmed in Calypso)
- Change the year via the date bar.
  - Verify the data updates accordingly. (can be compared with Calypso for accuracy)
- Verify the date bar only allows years the site has data for. (can be confirmed in Calypso)
  - Examples: 
    - en.blog should allow 2005 - 2019. 
    - Hogwarts should allow 2016 - 2019.

Update release notes:


<img width="400" alt="annual_details_2" src="https://user-images.githubusercontent.com/1816888/59880895-6e21bf80-936b-11e9-9f6b-c2293fb4181c.png">

<img width="400" alt="annual_details_1" src="https://user-images.githubusercontent.com/1816888/59880903-737f0a00-936b-11e9-9e04-428b9ba666d0.png">

<img width="400" alt="annual_details_3" src="https://user-images.githubusercontent.com/1816888/59880944-8f82ab80-936b-11e9-96de-ca542d020efc.png">
